classes/class-wc-connect-taxjar-control.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+class WC_Connect_TaxJar_Control {
+
+	/**
+	 * @var WC_Connect_API_Client
+	 */
+	public $api_client;
+
+	/**
+	 * @var WC_Connect_Logger
+	 */
+	public $logger;
+
+	const PROXY_PATH               = 'taxjar/v2';
+	const SETUP_WIZARD_OPTION_NAME = 'woocommerce_setup_automated_taxes';
+
+	public function __construct(
+		WC_Connect_API_Client $api_client,
+		WC_Connect_Logger $logger
+	) {
+		$this->api_client = $api_client;
+		$this->logger = $logger;
+	}
+
+	public function init() {
+		if ( ! class_exists( 'WC_Taxjar' ) ) {
+			return;
```

Comment:
How are we going to handle this? Right now, the TaxJar plugin isn't auto-installed in the Onboarding Wizard. With this change, the TaxJar integration would break for existing users.

I'm ok handling this in a different PR, but I just want to point out that we can't release this change as it is now.